blueoil/tfds_data_processor.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import pprint
+import tensorflow as tf
+
+
+class TFSequence:
```

Comment:
`TFSequence` name is looks like `tf.keras.Sequential`.
It's good to include `TFDS` in it's name.
How about `TFDSProcessorSequence` ? Too long?

Reply:
Yeah, I agree it is confusing. I will change to `TFDSProcessorSequence`!